src/ch/ch_conf.c

```diff
@@ -195,7 +195,7 @@ chExtractVersionInfo(int *retversion)
     if ((tmp = STRSKIP(tmp, "cloud-hypervisor v")) == NULL)
         goto cleanup;
 
-    if (virParseVersionString(tmp, &version, true) < 0)
+    if (virCHVersionString(tmp, &version, true) < 0)
```

Comment:
Does it even proceed to here, if `cloud-hypervisor --version` outputs `cloud-hypervisor msft/v`? It looks like it wolud fail right away at the condition above?

Thanks

Reply:
yes you are correct. Please check new push